Pod/Classes/TSMessage.m

```diff
@@ -248,7 +226,13 @@ - (void)fadeInCurrentNotification
         {
             navigationbarBottomOfViewController = [self.delegate messageLocationOfMessageView:currentView];
         }
-        
+
+        CGFloat verticalOffset = [[currentView.viewController topLayoutGuide] length];
+      
+        if ([currentView.viewController isKindOfClass:[UINavigationController class]]) {
+          verticalOffset += ((UINavigationController *)currentView.viewController).toolbar.frame.size.height;
```

Comment:
While this works, it doesn't adapt when the top layout guide changes (such as if the user changes the orientation of their device).